platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/types/FormattedSection.java

```diff
@@ -0,0 +1,84 @@
+package com.mapbox.mapboxsdk.style.types;
+
+import android.support.annotation.Keep;
+import android.support.annotation.Nullable;
+
+import java.util.Arrays;
+
+/**
+ * A component of the {@link Formatted}.
+ */
+@Keep
+public class FormattedSection {
+  private String text;
+  private double fontScale;
+  private String[] fontStack;
+
+  /**
+   * Creates a formatted section.
+   *
+   * @param text      displayed string
+   * @param fontScale scale of the font, 1.0 is default
```

Comment:
This works as an equivalent but as you pointed out it might not be visible right away. Since this PR doesn't add an option to use `Formatted` as a `text-font` argument, I'm going to make the constructor package-private (as it's called only from JNI) for the time being and expose it in a follow-up PR, making it clearer at the same time.